bitnami/kube-prometheus/Chart.yaml

```
@@ -35,4 +35,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/prometheus
   - https://github.com/bitnami/containers/tree/main/bitnami/alertmanager
   - https://github.com/prometheus-operator/kube-prometheus
-version: 8.4.1
+version: 8.6.0
```

Bumped 2 minors here

```suggestion
version: 8.5.0
```